Copyboard/AppDelegate.swift

```diff
@@ -68,6 +74,31 @@ class AppDelegate: NSObject, NSApplicationDelegate {
 			)
 		}
 	}
+	
+	@MainActor
+	private func _installPanelFocusObservers() {
+		guard let window = menuBar?.statusItem.window else { return };
+
+		NotificationCenter.default.addObserver(
+			forName: NSWindow.didBecomeKeyNotification,
+			object: window,
+			queue: .main
+		) { [weak self] _ in
+			Task { @MainActor in
+				self?._applyFocusPreference();
+			}
+		}
+	}
+	
+	@MainActor
+	private func _applyFocusPreference() {
```

Comment:
instead, we can always focus the search field instead of it being behind an option